CodeEdit/Features/UtilityArea/DebugUtility/TaskOutputActionsView.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  TaskOutputActionsView.swift
+//  CodeEdit
+//
+//  Created by Tommy Ludwig on 27.06.24.
+//
+
+import SwiftUI
+
+struct TaskOutputActionsView: View {
+    @ObservedObject var activeTask: CEActiveTask
+    @ObservedObject var taskManager: TaskManager
+    @Binding var scrollProxy: ScrollViewProxy?
+
+    @Namespace var bottomID
+    var body: some View {
+        HStack {
+            Spacer()
+
+            Button {
+                taskManager.runTask(task: activeTask.task)
```

Comment:
If the task is still running when this is pressed, it marks it as failed because it is terminated before starting again. Could we modify this slightly to mark the task as `notRunning` if the task is running?

This also causes a small issue where the Stop and Suspend/Resume buttons are disabled after a long-running task is restarted using this button. Because the task is marked "Failed" those buttons are disabled.

Reply:
Can you give it another shot? I handled exit code 15 separately from 1.